wrapper/CSharp/wolfCrypt-Test/wolfCrypt-Test.cs

```diff
@@ -1,928 +1,928 @@
-        Console.WriteLine("Testing hash cleanup...");
-        int freeResult = wolfcrypt.HashFree(hash, hashType);
-        if (freeResult != 0)
-        {
-            Console.WriteLine($"HashFree failed for {hashTypeName}");
-        }
-        else
-        {
-            Console.WriteLine("Hash cleanup test passed.");
-        }
-    } /* END hash_test */
-
-    public static void standard_log(int lvl, StringBuilder msg)
-    {
-        Console.WriteLine(msg);
-    }
-
-    public static void Main(string[] args)
-    {
-        try
-        {
-            Console.WriteLine("Starting Cryptographic Tests...\n");
-
-            wolfcrypt.Init();
-
-            /* setup logging to stdout */
-            wolfcrypt.SetLogging(standard_log);
-
-            random_test();
-
-            Console.WriteLine("\nStarting ECC tests");
-
-            ecc_test("SHA256", 32); /* Uses SHA-256 (32 byte hash) */
-            ecc_test("SHA384", 32); /* Uses SHA-384 (32 byte hash) */
-            ecc_test("SHA512", 32); /* Uses SHA-512 (32 byte hash) */
-
-            Console.WriteLine("\nStarting ECIES tests");
-
-            ecies_test(32); /* ECIES test (32 byte key size) */
-            ecies_test(48); /* ECIES test (48 byte key size) */
-            ecies_test(66); /* ECIES test (66 byte key size) */
-
-            Console.WriteLine("\nStarting ECDHE tests");
-
-            ecdhe_test(32); /* ECDHE shared secret test (32 byte key size) */
-            ecdhe_test(48); /* ECDHE shared secret test (48 byte key size) */
-            ecdhe_test(66); /* ECDHE shared secret test (66 byte key size) */
-
-            Console.WriteLine("\nStarting RSA tests");
-
-            rsa_test("SHA256", 2048); /* Uses SHA-256 (2048 bit hash) */
-            rsa_test("SHA384", 2048); /* Uses SHA-384 (2048 bit hash) */
-            rsa_test("SHA512", 2048); /* Uses SHA-512 (2048 bit hash) */
-
-            Console.WriteLine("\nStarting ED25519 test");
-
-            ed25519_test(); /* ED25519 test */
-
-            Console.WriteLine("\nStarting curve25519 test");
-
-            curve25519_test(); /* curve25519 shared secret test */
-
-            Console.WriteLine("\nStarting AES-GCM test");
-
-            aes_gcm_test(); /* AES_GCM test */
-
-            Console.WriteLine("\nStarting HASH tests");
-
-            hash_test((uint)wolfcrypt.hashType.WC_HASH_TYPE_SHA256); /* SHA-256 HASH test */
-            hash_test((uint)wolfcrypt.hashType.WC_HASH_TYPE_SHA384); /* SHA-384 HASH test */
-            hash_test((uint)wolfcrypt.hashType.WC_HASH_TYPE_SHA512); /* SHA-512 HASH test */
-            hash_test((uint)wolfcrypt.hashType.WC_HASH_TYPE_SHA3_256); /* SHA3_256 HASH test */
-
-            wolfcrypt.Cleanup();
-
-            Console.WriteLine("\nAll tests completed successfully");
-        }
-        catch (Exception ex)
-        {
-            Console.WriteLine($"An error occurred: {ex.Message}");
-            Environment.Exit(-1);
-        }
-    }
-}
+/* wolfCrypt-Test.cs

```

Comment:
This file shows a full replace. Guessing a CRLF issue. Please try and avoid that change if possible.

Reply:
Are you saying this file specifically is different? If so go ahead and keep it.